docs/micro-rdk/motor/gpio.md

```diff
@@ -0,0 +1,151 @@
+Find more information in the [encoded motor documentation](/components/motor/gpio/encoded-motor/).
+
+To configure a DC motor as a component of your robot, first configure the [board](/components/board/) to which the motor driver is wired.
+Then add your motor:
+
+{{< tabs name="gpio-config">}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `motor` type, then select the `gpio` model.
+Enter a name for your motor and click **Create**.
+
+![G P I O motor config in the builder UI with the In1 and In2 pins configured and the PWM pin field left blank.](/components/motor/gpio-config-ui.png)
+
+Edit and fill in the attributes as applicable.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json
+{
+  "components": [
+    {
+      "name": "<your-board-name>",
+      "model": "<your-board-model>",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+    },
+    {
+      "name": "<your-motor-name>",
+      "model": "gpio",
+      "type": "motor",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": {
+          <...>
+        },
+        "board": "<your-board-name>",
+        "max_rpm": <int>,
+        "min_power_pct": <float>,
+        "max_power_pct": <float>,
+        "pwm_freq": <float>,
+        "dir_flip": <float>,
+        "en_low": <float>
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+An example configuration for a `gpio` motor:
+
+```json
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "pi",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": []
+    },
+    {
+      "name": "example-gpio",
+      "model": "gpio",
+      "type": "motor",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": {
+          "dir": "36",
+          "pwm": "32"
+        },
+        "board": "local",
+        "max_rpm": 500
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for `gpio` motors:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `board` | string | **Required** | `name` of the [board](/components/board/) to which the motor driver is wired. |
+| `max_rpm` | int | **Required** | This is an estimate of the maximum RPM the motor will run at with full power under no load. The [`GoFor`](/components/motor/#gofor) method calculates how much power to send to the motor as a percentage of `max_rpm`. If unknown, you can set it to 100, which will mean that giving 40 as the `rpm` argument to `GoFor` or `GoTo` will set it to 40% speed. ***Not required** or available for [encoded](/components/motor/gpio/encoded-motor/) `gpio` motors.* |
```

Comment:
can we actually remove "max_rpm"? GoFor is not supported anyway and I think having this attribute as required is confusing. I'll make it so that micro-RDK doesn't expect it right now

Reply:
Done, removed